exporter/exporter.go

```diff
@@ -225,7 +236,15 @@ func (ne *NATSExporter) InitializeCollectors() error {
 		default:
 			return fmt.Errorf("invalid jsz filter %q", opts.GetJszFilter)
 		}
-		ne.createCollector(collector.JetStreamSystem, opts.GetJszFilter)
+		splitOrEmpty := func(s string) []string {
+			if s == "" {
+				return []string{}
+			}
+			return strings.Split(s, ",")
+		}
+		streamMetaKeys := splitOrEmpty(opts.JszSteamMetaKeys)
+		consumerMetaKeys := splitOrEmpty(opts.JszConsumerMetaKeys)
+		ne.createJszCollector(opts.GetJszFilter, streamMetaKeys, consumerMetaKeys)
```

Comment:
It may be smart to validate that the resulting label names are actually valid in Prometheus. WDYT?